content/news/050/index.md

```diff
@@ -76,56 +101,39 @@ If needed, a section can be split into subsections with a "------" delimiter.
 [Way of Rhea][wor] just got a release date: it will be coming to Steam on
 **May 20th, 2024**!
 
-Way of Rhea is a color based puzzle game with difficult puzzles, but forgiving
-mechanics being developed by [@masonremaley] in a custom Rust engine.
+Way of Rhea ([Steam](https://store.steampowered.com/app/1110620/Way_of_Rhea/)) is a color based puzzle game with difficult puzzles, but forgiving
+mechanics being developed by [@masonremaley] in a custom engine.
 
 You can support development by [wishlisting Way of Rhea on Steam][wor], or
 [signing up for the mailing list][wor-mail].
 
-Recent updates:
+The most important updates are:
```

Comment:
nit: wordy way of saying the same thing I think?

update: seems like this wording is because items are being removed (addressing that in another comment). I think staying on "Recent updates" makes more sense still.

Reply:
[`44c261b` (#1496)](https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1496/commits/44c261b5cd65f664854a03a9ead0951a1d145f9a)